config.json.example

```diff
@@ -3,5 +3,9 @@
   "hidserv": {
     "hostname": "avgfxawdn6c3coe3.onion",
     "key": "Il8Xas7uf6rjtc0LxYwhrx"
-  }
+  },
+  "vmsizes": {
```

Comment:
No, because it's JSON. =/ If/when this file becomes YAML, then we can add comments. The format question is best addressed in the context of #51, so recommend tolerating the lack of comments for now.